bio/prinseq-plus-plus/wrapper.py

```
@@ -0,0 +1,73 @@
+
+log = snakemake.log_fmt_shell(stdout=True, stderr=True, append=False)
+extra = snakemake.params.get("extra", "")
+
+
+# Input files
+input_pe = len(snakemake.input) == 2
+file2 = f"-fastq2 {snakemake.input[1]}" if input_pe else ""
+
+
+# Input format
+in_fmt = Path(snakemake.input[0].removesuffix(".gz")).suffix
```

Ruff

Undefined name `snakemake`

(F821)